src/main/java/org/jenkinsci/plugins/githubautostatus/BuildStatus.java

```diff
@@ -39,8 +39,7 @@
  */
 public class BuildStatus {
     
-    public BuildStatus(GHRepository repository, String shaString, String targetUrl, String context) {
-        this.repository = repository;
+    public BuildStatus(String shaString, String targetUrl, String context) {
```

Comment:
this breaks binary compatibility. Not sure whether the plugin retains it (up to the maintainers)

Reply:
Thanks for pointing it out. I think it's unlikely anyone else uses it. This plugin is only a couple months old, and it doesn't seem like it adds anything much on top of the base notification, so it should be OK. I did do a search of jenkinsci and didn't find anything other than this plugin referencing the changed classes, so at least nothing hosted there uses it.